### PR TITLE
ft: edit a sellers store

### PR DESCRIPTION
**Pull request by aapetsi**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project

### CODE REVIEW

**Comment by aapetsi** [comment]
[API Endpoint for sellers to edit account/store details](https://trello.com/c/YpfRrxy2/16-api-endpoint-for-sellers-to-edit-account-store-details)

**Review by kip-guile — APPROVED** [review]
All looks fine to me. Should work.

**Review by shaunorpen — APPROVED** [review]
Look good to me!

File: routes/authRouter.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 // @route POST api/auth/register
 // @desc Register new user
 // @access Public
 router.post('/register', sellerController.register)
 router.post('/login', sellerController.login)
 
+// @route POST api/auth/edit
+// @desc Edit a seller's store
+// @access Public
+router.put('/stores', authentication, editStore)
```
Comment (DimejiAre) [inline routes/authRouter.js]: @aapetsi endpoint for editing sellers should not be in authRouter.js

File: middlewares/authentication.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken')
```
Comment (DimejiAre) [inline middlewares/authentication.js]: @aapetsi The folder name is middleware on develop. This functionality also already exists in develop

**Review by DimejiAre — CHANGES_REQUESTED** [review]

File: middlewares/authentication.js (hunk 2 above)
Comment (aapetsi) [reply middlewares/authentication.js]: yes. i did that before Tolu pushed his up. I've fixed that conflict

File: routes/authRouter.js (hunk 1 above)
Comment (aapetsi) [reply routes/authRouter.js]: Yes I realised. Tolu suggested we have a folder clean up once everything has been merged

**Review by DimejiAre — APPROVED** [review]
@aapetsi Nice one. Looks good now, but it seems to be failing the checks.

**Comment by aapetsi** [comment]
> @aapetsi Nice one. Looks good now, but it seems to be failing the checks.
i've fixed the tests @DimejiAre

**Pull request merged**